frontend/src/Hooks/useModal.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState } from "react";
+
+export default function useModal() {
```

Comment:
This hook could be moved to Navbar, instead of having a separate file since it's a pretty simple hook.